src/cpp/fastdds/core/policy/QosPoliciesSerializer.hpp

```diff
@@ -63,6 +63,22 @@ class QosPoliciesSerializer
         return ParameterSerializer<QosPolicy>::cdr_serialized_size(qos_policy);
     }
 
+    static bool should_be_sent(
```

Comment:
In order to distinguish reader from writer here we could add a `const EntityId& entity_id`